AWS/Old_Generation_RDS_instances.ipynb

```diff
@@ -0,0 +1,734 @@
+    },
+    "id": 1,
+    "index": 1,
+    "inputschema": [
+     {
+      "properties": {},
+      "title": "aws_list_all_regions",
+      "type": "object"
+     }
+    ],
+    "jupyter": {
+     "source_hidden": true
+    },
+    "language": "python",
+    "legotype": "LEGO_TYPE_AWS",
+    "name": "AWS List All Regions",
+    "orderProperties": [],
+    "outputParams": {
+     "output_name": "regions",
+     "output_name_enabled": true,
+     "output_runbook_enabled": false,
+     "output_runbook_name": ""
+    },
+    "printOutput": true,
+    "startcondition": "not region",
+    "tags": [],
+    "uuid": "708ea4af5f8fe7096a15b3a52c4a657606bab9e177386fad7a847341ed607d64",
+    "version": "1.0.0"
+   },
+   "outputs": [],
+   "source": [
+    "#\n",
+    "# Copyright (c) 2021 unSkript.com\n",
+    "# All rights reserved.\n",
+    "#\n",
+    "\n",
+    "from pydantic import BaseModel, Field\n",
+    "from typing import Dict, List\n",
+    "import pprint\n",
+    "\n",
+    "from beartype import beartype\n",
+    "@beartype\n",
+    "def aws_list_all_regions_printer(output):\n",
+    "    if output is None:\n",
+    "        return\n",
+    "    pprint.pprint(output)\n",
+    "\n",
+    "\n",
+    "@beartype\n",
+    "def aws_list_all_regions(handle) -> List:\n",
+    "    \"\"\"aws_list_all_regions lists all the AWS regions\n",
+    "\n",
+    "        :type handle: object\n",
+    "        :param handle: Object returned from Task Validate\n",
+    "\n",
+    "        :rtype: Result List of result\n",
+    "    \"\"\"\n",
+    "\n",
+    "    result = handle.aws_cli_command(\"aws ec2 --region us-west-2 describe-regions --all-regions --query 'Regions[].{Name:RegionName}' --output text\")\n",
+    "    if result is None or result.returncode != 0:\n",
+    "        print(\"Error while executing command : {}\".format(result))\n",
+    "        return str()\n",
+    "    result_op = list(result.stdout.split(\"\\n\"))\n",
+    "    list_region = [x for x in result_op if x != '']\n",
+    "    return list_region\n",
+    "\n",
+    "\n",
+    "task = Task(Workflow())\n",
+    "task.configure(conditionsJson='''{\n",
+    "    \"condition_enabled\": true,\n",
+    "    \"condition_cfg\": \"not region\",\n",
+    "    \"condition_result\": true\n",
+    "    }''')\n",
+    "\n",
+    "task.configure(outputName=\"regions\")\n",
+    "task.configure(printOutput=True)\n",
+    "(err, hdl, args) = task.validate(vars=vars())\n",
+    "if err is None:\n",
+    "    task.execute(aws_list_all_regions, lego_printer=aws_list_all_regions_printer, hdl=hdl, args=args)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "dbbf73ef-3c3e-49b7-8c4b-301e02614d84",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false
+    },
+    "name": "Step-1",
+    "orderProperties": [],
+    "tags": [],
+    "title": "Step-1"
+   },
+   "source": [
+    "<h3 id=\"Filter-Unattached-EBS-Volumes\">AWS Get Older Generation RDS Instances</h3>\n",
```

Comment:
Is this legacy code that did not get updated?? <h3 id=\"Filter-Unattached-EBS-Volumes\">